src/detect-distance.c

```diff
@@ -160,55 +160,31 @@ static int DetectDistanceSetup (DetectEngineCtx *de_ctx, Signature *s,
 
 static int DetectDistanceTest01(void)
 {
-    int result = 0;
-
     DetectEngineCtx *de_ctx = DetectEngineCtxInit();
-    if (de_ctx == NULL) {
-        printf("no de_ctx: ");
-        goto end;
-    }
+    FAIL_IF_NULL(de_ctx);
 
     de_ctx->flags |= DE_QUIET;
 
     de_ctx->sig_list = SigInit(de_ctx, "alert tcp any any -> any any (content:\"|AA BB|\"; content:\"|CC DD EE FF 00 11 22 33 44 55 66 77 88 99 AA BB CC DD EE|\"; distance: 4; within: 19; sid:1; rev:1;)");
```

Comment:
While changing these tests we would like to change `SigInit` to `DetectEngineAppendSig`. See other tests for how this is different.